### PR TITLE
Introducing cs-lib folder for common-sense package

### PR DESCRIPTION
**Pull request by fredsonaguiar**
I've started introducing some fundamental axioms in cs-lib in a file named tiny.lean, based on kif/tptp output file tiny+spatial.

### CODE REVIEW

File: cs-lib/leanpkg.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[package]
+name = "cs-lib"
+version = "0.1"
+lean_version = "leanprover-community/lean:3.9.0"
+path = "src"
+
+[dependencies]
+mathlib = {git = "https://github.com/leanprover-community/mathlib", rev = "b14a26ee31a482c5f3d26069876481988b2c746f"}
```
Comment (arademaker) [inline cs-lib/leanpkg.toml:8]: do we really need the mathlib dependency?

File: cs-lib/src/spatial-reasoning-lib.lean [hunk 2]
```diff
@@ -1,21 +1,5 @@
-/- here we add some recurrent, usefull axioms -/
+import	tiny.lean
```
Comment (arademaker) [inline cs-lib/src/spatial-reasoning-lib.lean:1]: great!

**Comment by arademaker** [comment]
OK. Next, we will need one example of a problem that uses the axioms and constants defined in the lib. Now I understood the point about tiny+spatial. In SUMO @apease uses 

https://github.com/ontologyportal/sumo/blob/spatial/spatial.kif and https://github.com/ontologyportal/sumo/blob/spatial/tinySUMO.kif. 

We can try to reflect it here. The tiny+spatial.tptp was a name that I invented to the output of the transformation of these two KIF files to TPTP.

BTW, I didn't understand the difference between `tinySUMO.kif` and `Merge.kif` yet. I would expect to find in `Merge.kif` the minimal SUMO axioms that define the top-level ontology and the KIF language syntax and semantics.

**Comment by apease** [comment]
tinySUMO is just a tiny portion (a subset) of the Merge.kif file from SUMO that I use for testing inferences.  SUO-KIF is not defined in SUMO.  It's just the seven logical operators with a LISP-y syntax.  SUMO is built on top of that minimal language.  But things that one might expect to be built in to some other language, like the notion of a relation, or instance and class defining operators, are defined in SUMO, rather than specified in SUO-KIF

**Comment by arademaker** [comment]
Hi @apease , happy that you commented here. Let me put it another way. If there is a minimal set of axioms from `Merge.kif` that could be considered an essential part of SUMO necessary for the SigmaKEE's transformations, maybe it would make sense to extract it from `Merge.kif` to `tiny.kif` and make `Merge.kif` be the remaining set of axioms not in `tiny.kif`. That approach would make the organization much more clear and we would not have unnecessary copies of axioms in `tinySUMO.kif` and `Merge.kif`. Does it make sense?

Regarding the SUO-KIF language, I was not clear and you are of course right. SUO-KIF as a language is defined in https://github.com/ontologyportal/sigmakee/blob/master/suo-kif.pdf.  But maybe we can call its `semantics` the terms you listed (instance, classes, relations etc). If we agree on that, this is defined in SUMO.

**Comment by apease** [comment]
I agree that separating out the most minimal set of concepts and axioms to run Sigma from Merge.kif would be a good idea.  We sort of did that from the beginning with the different demarcated sections of the file (base ontology, objects etc), but a separate file would be better.  The semantics of SUO-KIF are really just the truth tables for the logical operators, and stand alone from the semantics of additional fundamental things like class and instance operators

**Pull request merged**